sensor-things-api-app/src/main/resources/application.yml

```diff
@@ -42,6 +42,9 @@ spring:
 # MQTT Configuration
 mqtt:
   broker:
+    plain_tcp:
+      enabled: false
```

Comment:
Should not this be enabled?

Reply:
As this configuration needs to be customized anyway for each deployment (e.g. change database access), it does not really matter what the default value is.

I set it to false to increase startup time by a fraction, but this is benefit is marginal and wen can ofc change it to be enabled by default.